pkg/storaged/index.html

```diff
@@ -762,6 +762,29 @@ <h4 class="modal-title">{{Title}}</h4>
                   <input class="form-control" type="text" {{#ValueMB}}value="{{.}}"{{/ValueMB}} data-field={{.}}>
                 </td>
                 {{/SizeInput}}
+                {{#SizeSlider}}
+                <td class="top">{{Title}}</td>
+                <td>
+                  <div class="dialog-wrapper">
+                    <div class="size-slider" data-field={{.}}>
+                      <table>
+                        <tr>
+                          <td>
+                            <div class="slider"></div>
+                          </td>
+                          <td>
+                            <input class="form-control size-text">
+                          </td>
+                          <td>
+                            <select class="form-control selectpicker size-unit">
```

Comment:
Would it be possible to avoid additional use of bootstrap-select, especially in templates? It's something we're trying to get rid of. playground/patterns.html includes an example of how to avoid it. Or is that too hard to do here?